feature-libs/product-configurator/rulebased/components/attribute/header/configurator-attribute-header.component.ts

```diff
@@ -137,4 +144,41 @@ export class ConfiguratorAttributeHeaderComponent
     const images = this.attribute.images;
     return images && this.hasImage ? images[0] : undefined;
   }
+
+  navigateToGroup(): void {
+    this.configuratorCommonsService
+      .getConfiguration(this.owner)
+      .pipe(take(1))
+      .subscribe((configuration) => {
+        if (this.groupType === Configurator.GroupType.CONFLICT_GROUP) {
+          const groupId = this.attribute?.groupId;
+          if (groupId) {
```

Comment:
What if group is undefined here? Under which circumstances does that happen? (Not covered by test)

Reply:
Done!